src/components/utility/Typography/Typography.module.css

```diff
@@ -1,58 +1,48 @@
-/* defined in App.css:
-@import url('https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,600;0,700;1,600;1,700&display=swap');
-
-body {
-  font-family: "Livvic", -apple-system, BlinkMacSystemFont, 'Segoe UI',
-    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
-    'Helvetica Neue', sans-serif;
-  font-weight: 600;
-}
-*/
-
-h1, h2, h3 {
+h1,
+h2,
+h3 {
   font-weight: 700; /* bold */
 }
 
-h1, h1.h1Large {
-  font-size: 64px;
-  line-height: 56px;
+h1 {
+  font-size: 2.5rem;
+  line-height: 2.5rem;
 }
 
 h2 {
-  font-size: 32px;
-  line-height: 32px;
+  font-size: 2rem;
+  line-height: 2rem;
 }
 
-h3 {
-  font-size: 18px;
-  line-height: 28px;
+h3 .body1 {
+  font-size: 0.9rem;
+  line-height: 1.75rem;
 }
 
-.body1 {
-  font-size: 18px;
-  line-height: 28px;
+.body2 {
+  font-size: 0.9rem;
+  line-height: 1.5rem;
 }
 
-.body2 {
-  font-size: 15px;
-  line-height: 25px;
+@media (min-width: 768px) {
+  h1 {
+    font-size: 4rem;
+    line-height: 3.5rem;
+  }
 }
 
-@media (width >= 960px) {
-  h1.h1Large {
```

Comment:
this class should not be removed but I have to run off before I can finish explaining why; I'll come back to this in more detail

Reply:
There are two distinct styles for the `h1` element, because the home page header ("Find the best talent") is extra-large on desktops, but is the same as all the other page headers otherwise ("About" and "Contact").

If you want to give it a different elType to distinguish it (maybe "hero" instead of a subtype of h1?), I could see that. But I don't think fully removing it is correct.

Reply:
I made them h1Large and h1Small as earlier